### PR TITLE
feat(navigatie): rename menu to navigation

### PR DESCRIPTION
**Pull request by bertyhell**

### CODE REVIEW

**Review by WardVerc1 — APPROVED** [review]

File: src/admin/navigations/views/NavigationBarDetail.tsx [hunk 1]
```diff
@@ -7,17 +7,17 @@ import { GENERATE_SITE_TITLE } from '../../../constants';
...
+import './NavigationBarDetail.scss';
 
-type MenuDetailProps = DefaultSecureRouteProps<{ menu: string }>;
+type NavigationBarDetailProps = DefaultSecureRouteProps<{ menu: string }>;
 
-const MenuDetail: FunctionComponent<MenuDetailProps> = ({ match }) => {
+const NavigationBarDetail: FunctionComponent<NavigationBarDetailProps> = ({ match }) => {
 	const { tText } = useTranslation();
 
 	const menuId = match.params.menu;
```
Comment (silkederudder-shd) [inline src/admin/navigations/views/NavigationBarDetail.tsx:17]: I'd refactor this to `navigationId` for consistency

File: src/admin/navigations/navigations.const.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { ROUTE_PARTS } from '../../shared/constants';
+
+export const NAVIGATIONS_PATH = {
+	NAVIGATIONS_OVERVIEW: `/${ROUTE_PARTS.admin}/${ROUTE_PARTS.menu}`,
```
Comment (silkederudder-shd) [inline src/admin/navigations/navigations.const.ts:4]: Maybe refactor `ROUTE_PARTS.menu` to `ROUTE_PARTS.navigation`

File: src/admin/navigations/views/NavigationBarDetail.scss [hunk 3]
```diff
@@ -0,0 +1,60 @@
+@import '../../../styles/settings/colors';
+@import '../../../styles/settings/variables';
+
+// Elements
+.c-menu-detail__table {
```
Comment (silkederudder-shd) [inline src/admin/navigations/views/NavigationBarDetail.scss:5]: `.c-navigation...`

**Review by silkederudder-shd — CHANGES_REQUESTED** [review]

File: src/admin/navigations/views/NavigationBarDetail.tsx (hunk 1 above)
Comment (bertyhell) [reply src/admin/navigations/views/NavigationBarDetail.tsx:17]: fixed in commit on pr higher on the stack:
https://github.com/viaacode/avo2-client/pull/1683/commits/1084917d5c4b842be4ff0d47fac51a268eafa026

File: src/admin/navigations/navigations.const.ts (hunk 2 above)
Comment (bertyhell) [reply src/admin/navigations/navigations.const.ts:4]: fixed in commit on pr higher on the stack:
https://github.com/viaacode/avo2-client/pull/1683/commits/1084917d5c4b842be4ff0d47fac51a268eafa026

File: src/admin/navigations/views/NavigationBarDetail.scss (hunk 3 above)
Comment (bertyhell) [reply src/admin/navigations/views/NavigationBarDetail.scss:5]: fixed in commit on pr higher on the stack:
https://github.com/viaacode/avo2-client/pull/1683/commits/1084917d5c4b842be4ff0d47fac51a268eafa026
https://github.com/viaacode/react-admin-core-module/pull/89/commits/ae3129a4a30f123b1a4821203adca7267c366b6c

**Pull request merged**